www/index.html

```diff
@@ -200,7 +200,7 @@
                     <a href="/matrix/" class="col-50 open-matrix-pb">
                       <div class="nollning-btn-matrix"></div>
                     </a>
-                    <a href="https://www.fsektionen.se/anonym" target="_blank" class="col-33 external">
+                    <a href="/introduction_contacts/" class="col-50 open-matrix-pb">
```

Comment:
This should be a 33% column
```suggestion
                    <a href="/introduction_contacts/" class="col-33">
```